cypress/integration/ClientCreation.js

```diff
@@ -0,0 +1,22 @@
+import ClientsPage from '../helpers/ClientsPage'
+
+// T-4
+describe('Client creation', () => {
+
+    it('Admin should be able to create clients', () => {
+        cy.loginAs('Admin')
+        ClientsPage.visit()
+
+        const randomName = Math.random().toString(36).substring(9);
+
+        ClientsPage.getCreateButton().click()
+
+        ClientsPage.getOrganizationField().type(randomName)
+        ClientsPage.getFirstNameField().type('first name')
+        ClientsPage.getLastNameField().type('Last name')
+        ClientsPage.getEmailField().type('my@email.com')
+        ClientsPage.getSaveButton().click()
```

Comment:
It doesn't look like that you check that client you just created is actually created

Reply:
I believe it's fixed.

Reply:
Don't see any assertions for created client